QUESTIONS.md

```diff
@@ -3,3 +3,20 @@
 What questions can you come up with?
 
 ## Cohort Questions
+
+1. How can we better troubleshoot things in node_modules when it won't load or initialize installed packages?
+
+2. What are some ways I can manage burnout and paralysis analysis?
```

Comment:
Best thing to remember is not 1 person is leaving this cohort with 100% knowledge. Everyone will have strengths and weaknesses.

You will never (and no other developer ever does) start a medium+ sized project and finish it in one sitting knowing how to do everything from memory. The important thing is you gain the confidence to find tools to learn things on the fly and use them. Don't be afraid to fail at things, and just keep going until the damn code compiles. 

That being said, if you ever get a hint of burnout, just take a break. The code isn't going anywhere, you can finish it when you get back, and you may even hop back on and realize with a fresh set of eyes that the answer was right in front of you.